src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -260,4 +264,124 @@ public void validateRecordsTransferredToTargetTableIsEqualToNumberOfRecordsFromS
     BeforeActions.scenario.write("Number of records transferred from source table to target table:" + count);
     Assert.assertEquals(count, countRecordsTarget);
   }
+
+  @Then("Enter BigQuery source properties partitionFrom and partitionTo")
```

Comment:
No need to add the step definition. Please make use of existing steps. This was discussed may times in our calls.